pkg/auth/arn_test.go

```diff
@@ -73,6 +73,51 @@ func TestParseARN(t *testing.T) {
 	}
 }
 
+func TestParseResources(t *testing.T) {
+	cases := []struct {
```

Comment:
Please add additional cases to check that it's standard with the JSON marshal / unmarshal. Meaning literally use json.Marshal not hardcoded strings. 

- empty array `[]`
- non empty array 
- Invalid json format -> expect error